src/CPE/cpedict_priv.c

```diff
@@ -291,25 +288,9 @@ static void cpe_notes_export(const struct cpe_notes *notes, xmlTextWriterPtr wri
 
 struct cpe_notes *cpe_notes_new(void);
 void cpe_notes_free(struct cpe_notes *notes);
-/***************************************************************************/
-
-/* Add item to dictionary. Function that just check both variables
- * on NULL value.
- * */
-static bool cpe_dict_model_add_item(struct cpe_dict_model *dict, struct cpe_item *item)
-{
-
-	__attribute__nonnull__(dict);
-	__attribute__nonnull__(item);
-
-	if (dict == NULL || item == NULL)
-		return false;
-
-	oscap_list_add(dict->items, item);
-	return true;
-}
 
 /***************************************************************************/
+
```

Comment:
I didn't really do it on purpose, but it was how the code ended up after I deleted the function.  I was just balancing the blank lines above and below because I thought it looked nice.  We can change it if it bothers you.